README.rst

```diff
@@ -101,7 +101,41 @@ manner.
 
 - `workflow.yaml <workflow/yadage/workflow.yaml>`_
 
-**FIXME**
+Since yadage only accepts one input directory as parameter we are going to create a wrapper directory which will contain links to `inputs` and `outputs` directories:
+
+.. code-block:: console
+
+   $ mkdir -p yadage-inputs/code yadage-inputs/inputs
+   $ ln code/helloworld.py  yadage-inputs/code/helloworld.py
+   $ ln inputs/names.txt  yadage-inputs/inputs/names.txt
```

Comment:
Can be simplified to:

```
$ mkdir -p yadage-inputs/code yadage-inputs/inputs
$ ln code/helloworld.py  yadage-inputs/code/helloworld.py
$ ln inputs/names.txt yadage-inputs/inputs/
```
or, perhaps even better, since we don't do symbolic links, we could do:

```
$ mkdir yadage-inputs
$ cp -a code inputs yadage-inputs/
```
```